<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizStart</class>
 <widget class="QMainWindow" name="QuizStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>711</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QMainWindow {
    background-color: rgb(240, 240, 250); /* Light purple background */
}

QGroupBox {
    background-color: rgb(255, 255, 255); /* White background */
    border: 2px solid rgb(104, 90, 180); /* Purple border */
    border-radius: 10px;
    padding: 10px;
}

QLabel {
    font-size: 14pt;
    color: rgb(50, 30, 100); /* Dark purple text */
}

QLabel#questionLabel {
    font-size: 16pt;
    font-weight: bold;
    color: rgb(50, 30, 100); /* Dark purple */
}

QPushButton {
    background-color: rgb(104, 90, 180); /* Purple background */
    color: white;
    font-size: 12pt;
    padding: 6px 12px;
    border-radius: 5px;
}

QPushButton:hover {
    background-color: rgb(120, 110, 200); /* Lighter purple on hover */
}

QPushButton:pressed {
    background-color: rgb(80, 60, 160); /* Darker purple on press */
}

QPushButton#button50, QPushButton#buttonSwap, QPushButton#submit {
    background-color: rgb(104, 90, 180); /* Purple background */
}

QPushButton#button50:hover, QPushButton#buttonSwap:hover, QPushButton#submit:hover {
    background-color: rgb(120, 110, 200);
}


QRadioButton {
    font-size: 12pt;
    color: rgb(50, 30, 100); /* Dark purple */
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
    border: 2px solid rgb(104, 90, 180); /* Purple border for circle */
    border-radius: 10px;
    background-color: white;
}

QRadioButton::indicator:checked {
    background-color: rgb(104, 90, 180); /* Purple circle when checked */
}
/
QProgressBar {
    border: 1px solid rgb(104, 90, 180); /* Purple border */
    border-radius: 5px;
    background-color: rgb(240, 240, 250); /* Light purple background */
    height: 20px;
}

QProgressBar::chunk {
    background-color: rgb(104, 90, 180); /* Purple progress chunk */
    width: 1px;
}

QTableWidget {
    background-color: rgb(255, 255, 255); /* White background */
    border: 1px solid rgb(104, 90, 180); /* Purple border */
    gridline-color: rgb(104, 90, 180); /* Purple grid lines */
    font-size: 12pt;
    color: rgb(50, 30, 100); /* Dark purple text */
}

QTableWidget::horizontalHeader {
    background-color: rgb(104, 90, 180); /* Purple header */
    color: white;
    font-weight: bold;
    font-size: 14pt;
}

QPushButton#closeButton {
    background-color: rgb(104, 90, 180); /* Purple background */
    color: white;
    font-size: 12pt;
    padding: 6px 12px;
    border-radius: 5px;
}

QPushButton#closeButton:hover {
    background-color: rgb(120, 110, 200);
}

QPushButton#closeButton:pressed {
    background-color: rgb(80, 60, 160);
}

QMenuBar {
    background-color: rgb(104, 90, 180); /* Purple menu bar */
    color: white;
}

QStatusBar {
    background-color: rgb(240, 240, 250); /* Light purple */
    color: rgb(50, 30, 100); /* Dark purple text */
}

QLabel#scoreLabel, QLabel#rankLabel {
    font-size: 14pt;
    color: rgb(50, 30, 100); /* Dark purple */
}

QLabel#timeLabel {
    font-size: 12pt;
    color: rgb(50, 30, 100); /* Dark purple */
    font-weight: bold;
    background-color: rgb(240, 240, 250); /* Light purple background */
    padding: 5px 10px;
    border-radius: 5px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Starting Quiz</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>601</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">  border: 2px solid rgb(104, 90, 180); /* Purple border */</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700; text-decoration: underline;&quot;&gt;Rules for the User&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;1.In Game Mode, the quiz will end immediately after your first wrong answer.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;You have two lifelines, each usable only once:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;•50/50 removes two incorrect options.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;•Swap Question replaces the current question with a new one.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;2.In Test Mode, you must answer within the time limit.If the timer runs out, the quiz will end.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;3.In Practice Mode, there is no time limit and no penalties for wrong answers.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;4.Once you submit an answer, you cannot change it.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;5.Your score is based on how many questions you answer correctly.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>460</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>451</width>
       <height>94</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Topic:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_mode">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_topic">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_type">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
